packages/apps-config/src/api/spec/crust-parachain.ts

```diff
@@ -0,0 +1,76 @@
+// Copyright 2017-2021 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+// structs need to be in order
+/* eslint-disable sort-keys */
+
+export default {
+  Address: 'AccountId',
+  AddressInfo: 'Vec<u8>',
+  FileAlias: 'Vec<u8>',
+  Guarantee: {
+    targets: 'Vec<IndividualExposure<AccountId, Balance>>',
+    total: 'Compact<Balance>',
+    submitted_in: 'EraIndex',
+    suppressed: 'bool'
+  },
+  IASSig: 'Vec<u8>',
+  Identity: {
+    pub_key: 'Vec<u8>',
+    code: 'Vec<u8>'
+  },
+  ISVBody: 'Vec<u8>',
+  LookupSource: 'AccountId',
+  MerchantInfo: {
+    address: 'Vec<u8>',
+    storage_price: 'Balance',
+    file_map: 'Vec<(Vec<u8>, Vec<Hash>)>'
+  },
+  MerchantPunishment: {
+    success: 'EraIndex',
+    failed: 'EraIndex',
+    value: 'Balance'
+  },
+  MerkleRoot: 'Vec<u8>',
+  OrderStatus: {
+    _enum: ['Success', 'Failed', 'Pending']
+  },
+  PaymentLedger: {
+    total: 'Balance',
+    paid: 'Balance',
+    unreserved: 'Balance'
+  },
+  Pledge: {
+    total: 'Balance',
+    used: 'Balance'
+  },
+  ReportSlot: 'u64',
+  Releases: {
+    _enum: ['V1_0_0', 'V2_0_0']
+  },
```

Comment:
This should not the be needed - the base API version is comprehensive. But ok if you feel it is needed.